queries/thrift/highlights.scm

```diff
@@ -36,19 +36,22 @@
 ; Variables
 
 (identifier) @variable
-(annotation_identifier) @variable
+(enum_identifier) @variable
 
 ; Constants
 
 (const_identifier) @constant
-(enum_identifier) @constant
```

Comment:
I think this was fine as `@constant`.

Reply:
I redefined how they are parsed [here](https://github.com/amaanq/tree-sitter-thrift/commit/f4af461e2e40343e5b2fe628d41e8afe69a15944)

Reply:
So there is no `enum_identifier` now?

Reply:
enum Foo - Foo is now an enum_identifier instead of a generic type_identifier

```thrift
enum Numberz {
    One,
}

const Numberz myNumberz = Numberz.One;
```

Both One's are now enum_members which is more sound to me

Reply:
So…shouldn't it be `@type`?

Reply:
yep you're right, didn't think about that

Reply:
fixed